docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing BKPR
+
+A Bitnami Kubernetes Production Runtime (BKPR) release consists of a collection of Kubernetes manifests written in Jsonnet plus the accompanying `kubeprod` installer binary. The kubeprod binary deals with all the platform-specific details, evaluates the Jsonnet manifests, and applies them to the existing Kubernetes cluster.
+
+This document walks you through installing the `kubeprod` binary.
+
+## Installing the `kubeprod` installer
```

Comment:
"Installing the installer" sounds a bit weird, maybe just retitle to `Install kubeprod`?